README.md

```diff
@@ -21,31 +19,152 @@ func sayHelloTo(name: String) {
 }
 ```
 
-
-
 ## What the student can do at this point 
 
-* Create functions with one/two arguments, they haven't seen the function return anything yet.
+* Create functions with one/two arguments.
 
 ```swift
 func sayHelloTo(person: String) {
     print("Hello \(person)")
 }
 ```
 * Create variables and constants
-* Knows how to use the print function
-* Knows of the types String and Int
+* Use the `print` function
+* Know of the types `String` and `Int`
+
+# Returning Values From Functions
+
+So far, you've spent several lessons (and a couple of labs) learning how to create and call your own functions. You've seen that you can create a function that takes zero or more arguments and print a string (using Swift's `print` function) that interpolates those arguments.
+
+While it may look like functions are simply self-contained blocks of code that can print strings, they can actually do a lot more than that. They can also _return_ a new value.
+
+Imagine you want to write a function that can take the name of a friend, and tell you that friend's age. To keep this simple, we'll imagine that all your friends are 29 (since you should only have friends you can trust, and you shouldn't trust anyone over 30). How would you write that sort of function?
+
+Think back to when you wrote a function that took an argument. Let's review that by first writing a `printAge` function that prints "&lt;friend&gt; is 29" to the console. Add this to a new playground file:
+
+```swift
+func printAge(name: String) {
+    print("\(name) is 29")
+}
+```
+
+Let's review the parts of this function definition. First, you introduce a new function using the `func` keyword. Then you write the name of the function—in this case, `printAge`. Next comes the parameter list in parentheses. This function takes one `String` argument, a `name`. Then you open a set of curly braces and write the function body between them.
+
```

Comment:
This goes a long way in helping the students realize whether or not they really do understand the code snippets they're reading.